src/index.js

```diff
@@ -18,19 +18,38 @@ const SOURCE = `Promise.allSettled([p1, p2]).finally(() => {
 const CONFIG = [
   {
     plugins: [
-      [
-        "babel-plugin-polyfill-corejs3",
-        {
-          method: "usage-global",
-          targets: {
-            edge: 16,
-          },
-        },
-      ],
+      // [
+      //   "babel-plugin-polyfill-corejs3",
+      //   {
+      //     method: "usage-global",
+      //     targets: {
+      //       edge: 16,
+      //     },
+      //   },
+      // ],
+      // {
+      //   name: "babel-plugin-polyfill-corejs3",
+      //   description: "does this",
+      //   fileLocation: "babel-plugin-polyfill-corejs3",
+      //   defaultConfig: {
+      //     method: "usage-global",
+      //     targets: {
+      //       edge: 16,
+      //     }
+      //   },
+      // },
```

Comment:
Remove comments or describe what this is. Looks like this is trying to show what these values will be when the function runs.